tests/test_neb.py

```diff
@@ -134,15 +134,15 @@ def test_get_ts_guess_neb():
     reactant = Reactant(
         name="inital",
         charge=-1,
-        mult=0,
+        mult=1,
         solvent_name="water",
         atoms=xyz_file_to_atoms("sn2_init.xyz"),
     )
 
     product = Reactant(
         name="final",
         charge=-1,
-        mult=0,
```

Comment:
Sorry if I misunderstood, but `mult` in autodE previously meant total number of unpaired electrons, and now it has been changed to 2S+1, is that correct?

Reply:
was always 2S+1... lots of the tests were just very wrong!